lib/msal-common/test/account/AuthToken.spec.ts

```diff
@@ -77,8 +77,9 @@ describe("AuthToken.ts Class Unit Tests", () => {
         it("Throws error if rawIdToken is null or empty", () => {
             expect(() => new AuthToken("", cryptoInterface)).to.throw(ClientAuthErrorMessage.nullOrEmptyToken.desc);
             expect(() => new AuthToken("", cryptoInterface)).to.throw(ClientAuthError);
-
+            // @ts-ignore
```

Comment:
exception to function params being non-nullable?

Reply:
Can we just add an exception to tests?

Reply:
this is more to test vanilla JS scenarios. Typescript will catch them regardless. We are making some changes to our tests, one of them may be to test vanilla JS and typescript separately.

Reply:
> Can we just add an exception to tests?

Not sure what you mean.

Reply:
Allow null in function params for test files in lint rules.